setup.py

```diff
@@ -11,6 +11,7 @@
     'requests',
     'pyparsing',
     'scikit-learn',
+    'class_resolver',
```

Comment:
We cannot add `class-resolver` package back in since it is not available on anaconda.org. That's why we started writing our own solution for that (with a limited feature set): https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/resolver.py

Reply:
Hmm I can look into putting it in conda